doc/whats-new.md

```diff
@@ -1,356 +1,197 @@
-is that computing the full covariance matrix and all the epsilons at
-the same time was too memory intensive. Moreover, the data transfer of
-the epsilons from the master node to the workers was too big, except
-in toy models. This is why now only the cases of no correlation and
-full correlation are supported, in a lot more efficient way than they
-were supported before. No covariance matrix is needed, since the
-engine uses the standard normal distribution to generate the
-epsilons. It means that now it is possible to run much larger
-computations.
-
-The configuration parameter `asset_loss_table` has been removed. If you
-have a `job.ini` with that parameter, you will be warned that the
-parameter will be ignored.
-
-Also, a configuration parameter `ignore_covs` has been added. If the
-computation is so large that it does run out of memory even with the
-new improved mechanism, you can still run it by setting
-`ignore_covs=true` in the `job.ini` file: with this trick the epsilons
-will not be generated at all, even if your vulnerability functions
-have nonzero coefficients of variation. You will lose the effects of
-asset correlation, but an imperfect computation that runs has been
-deemed better than a perfect computation that does not run.
-
-Another big change is that the computation of loss curves has been moved in
-a postprocessing phase and it is a lot more efficient than before, especially
-in terms of data transfer. Also, loss maps are now dynamic outputs, i.e.
-they are computed only on demand at export time. Earlier, they would be
-computed and stored, even if they were not exported.
-
-[2] When the vulnerability functions have nonzero coefficients of variations,
-the engine (versions <= 2.1) computes a matrix of (A, E) floats (the epsilons)
-where A is the number of assets and E the number of seismic events.
-
-.npz exports
--------------
-
-We started using the `.npz` export format for numpy arrays. This is
-extremely convenient for Python applications, since an `.npz` file can be
-managed as a dictionary of arrays (just call `numpy.load('data.npz')`).
-In particular our QGIS plugin is able to read the exported `.npz` files
-and display things liks hazard curves, hazard maps and uniform hazard
-spectra. We also have a tool to convert `.npz` files into `.hdf5` files,
-if you prefer that format, just run
-
-```bash
-$ oq to_hdf5 *.npz
-```
-
-and all of your `.npz` files will be converted into `.hdf5` files.
-In the future a lot more outputs are expected to be exported into
-`.npz` format.
+The "Remove" button now actually removes the calculation. Before
+it was only hiding it from the Web UI. Be careful with it, since it is
+final. If you remove a calculation, the only way to restore it is
+to repeat the computation.
+
+We fixed the names of the files downloaded from the Web UI. They were
+containing slashes, that were then replaced in a browser-dependent way
+with dashes or underscores. This is solved now.
```

Comment:
The names of files downloaded from the Web UI previously contained slash ('/') characters that were then replaced with dashes or underscores by the browser.  We no longer include slash characters in file names.